src/status.ts

```diff
@@ -359,7 +361,11 @@ export class FormatterStatus extends StatusItem {
   }
 
   refresh(): void {
-    this.item.text = `Using formatter: ${this.client.formatter}`;
+    if (this.client.formatter) {
+      this.item.text = `Formatter: ${this.client.formatter}`;
+    } else {
+      this.item.text = `Formatter: upgrade server to v0.12.4 or higher`;
```

Comment:
I worry this could be interpreted as formatter won't work with versions before `v0.12.4`. How about:

```suggestion
      this.item.text = `Formatter: requires server to be v0.12.4 or higher to display this field`;
```